src/librustc_infer/infer/lexical_region_resolve/mod.rs

```diff
@@ -904,13 +914,13 @@ impl<'cx, 'tcx> LexicalResolver<'cx, 'tcx> {
         );
     }
 
-    fn collect_concrete_regions(
+    fn collect_bounding_regions(
```

Comment:
Pre-existing obviously but it'd be nice to have a comment with what the various things are in the returned tuple. I guess that the new set is the all nodes reachable from `orig_node_idx` in the given direction `dir`?